### ISSUE TITLE
U2F DEVICE_INELIGIBLE

### ISSUE THREAD

**Issue by justinvforvendetta** [LedgerHQ/ledgerjs#519]
can no longer sign into any services. was working yesterday, and now it doesn't work anywhere. no browser will accept it. have tried brave, firefox, chrome, edge, as well as multiple services. all the same response/error message.

i have seen this same issue on many repositories, with no one from ledger responding. this is a product we bought from your company. it's quite literally your job to respond to this with a solution, and i have seen no responses from this issue for a year now. people are having this problem for a year, and ledger offers no solution? that's bizarre. of course, not to mention, what happens if ledger company disappears? how would we retrieve our funds from these devices?

**Comment by lentils79** [comment]
With the seed phrase (which you should have backed up) you can recreate the wallet elsewhere should the need should arise.

**Comment by justinvforvendetta** [comment]
@lentils79 where is the tool that converts the 24 word seedphrase?

**Comment by lentils79** [comment]
> @lentils79 where is the tool that converts the 24 word seedphrase? 

You can use your ledger directly with the electrum wallet to access your btc even if ledger live is no longer supported. You can connect the ledger directly to a metamask wallet to access Eth and all ERC20 tokens. 

The ledger device itself will store the seed phrase.. no need to convert it to get private keys, the device handles the process of signing transactions. 

Alternatively...
github.com/iancoleman/bip39/releases

There's an offline version. Do not put your seed words into the online site. Recipe for disaster!!!

**Comment by justinvforvendetta** [comment]
ah yeah i have the bip39 tool.. didnt realize it supported 24 words.. it still sucks that ledger doesnt respond to/fix the problems that -most- users are having. they sell a very serious product, but dont take it very seriously.

**Comment by testkovan** [comment]
The solution to this issue (especially on Windows - other OS are likely still fine) is to use a different transport or ask the affected service to use a different transport. See https://www.ledger.com/windows-10-update-sunsetting-u2f-tunnel-transport-for-ledger-devices/

**Comment by gre** [comment]
Yes, unfortunately with U2F protocol, there is nothing we can easily fix. The protocol is very limited.

It is recommended to use webusb and/or webhid API for web integration.

I don't really understand what you want is to fix in this issue. Is it about documenting this error code? I tend to simply deprecate the u2f library..

**Comment by justinvforvendetta** [comment]
@gre i have been on almost every DeFi site that allows ledger, they all have this problem with U2F, even coinbase's DeFi site. Are there any that you are aware of that are using hw-transport-webusb that you are aware of?

**Comment by Ps3ud-0nym** [comment]
> Yes, unfortunately with U2F protocol, there is nothing we can easily fix. The protocol is very limited.
> 
> As @testkovan said,
> It is recommended to use webusb and/or webhid API for web integration.
> 
> I don't really understand what you want us to fix in this issue. Is it about documenting this error code?
> 
> In our too level readme is mentioned that u2f have issues on windows, our customer support team is also fully aware of this issue.
> I think we'll also more explicitly deprecate the u2f library to request more explicitly for integrations to switch..

Am I likely to have more luck using Ubuntu? If so I'll boot to a live OS and try that instead.
